Library/Formula/augeas.rb

```diff
@@ -3,6 +3,7 @@ class Augeas < Formula
   homepage "http://augeas.net"
   url "http://download.augeas.net/augeas-1.4.0.tar.gz"
   sha256 "659fae7ac229029e60a869a3b88c616cfd51cf2fba286cdfe3af3a052cb35b30"
+  revision 1
```

Comment:
Let's not force everyone to reinstall their local copies here. Anyone who wants the example program urgently can just `brew reinstall augeas`. Thanks.